apps/web/src/components/ImageSmart/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Image, { ImageProps, StaticImageData } from 'next/image';
```

Comment:
nit: rename this from `Smart` to something descriptive like `MaybeOptimizedImage` that describes what's smart about it.

Reply:
gonna use <ImageOptimized /> for now !

Reply:
`ImageOptimized` feels weird to me too because the extra functionality of the wrapper is that it *isnt* optimized in some cases